pkg/vm/engine/disttae/types.go

```diff
@@ -281,7 +281,51 @@ type emptyReader struct {
 type BlockMeta struct {
 	Rows    int64
 	Info    catalog.BlockInfo
-	Zonemap [][64]byte
+	Zonemap []Zonemap
```

Comment:
I think zonemap/blockmeta has been binary compatible and no longer need marshal/unmarshal code.

Reply:
I've tried to remove `marshal/unmarshal` code for BlockMeta, but many bvt test cases were failed.
I'm afraid that we must keep them.

And `type Zonemap [64]byte`, they are the same structure under the hood